openebs/openebs-provisioner.go

```diff
@@ -90,7 +93,12 @@ func (p *openEBSProvisioner) Provision(options controller.VolumeOptions) (*v1.Pe
 	volSize := options.PVC.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)]
 	volumeSpec.Metadata.Labels.Storage = volSize.String()
 
-	volumeSpec.Metadata.Labels.StorageClass = *options.PVC.Spec.StorageClassName
+	className := GetPersistentVolumeClass(options)
+
+	if *className == "" {
+		glog.Errorf("Volume has no storage class specified")
```

Comment:
I have a query here:: What exactly happens when glog.Errorf is invoked?

Reply:
Will above logging avoid the nil pointer exception that is supposed to happen below!!!

Reply:
if `className` is `nil` it will log it and proceed further.